presidio_evaluator/data_generator/presidio_pseudonymize.py

```diff
@@ -33,10 +31,10 @@ def __init__(self, map_to_presidio_entities: bool = True, **kwargs):
             self.add_provider_alias("date_time", "DATE_TIME")
 
     def pseudonymize(
-        self,
-        original_text: str,
-        presidio_response: List[RecognizerResult],
-        count: int,
+            self,
```

Comment:
Looks like over-indentation, is it?

Reply:
Depends on which linter is used I guess! Flake8 doesn't complain about it, but maybe hasn't been configured for this
This formatting came from PyCharm's default formatting settings
I don't know what rules it follows to make it do 3 tabs here rather than 2

Reply:
Yes, not sure if it would cause any issues, but just to be on the safe side, could you revert that one small piece?

Reply:
Sorted 👍